package.json

```diff
@@ -82,9 +83,10 @@
     "morgan": "^1.10.0",
     "nanoid": "^3.3.4",
     "nodemailer": "^6.7.8",
-    "pm2": "^5.2.0",
+    "pm2": "^5.3.0",
+    "request": "^2.88.2",
```

Comment:
what's the use of request package?

Reply:
provides central logging and help Node.js application running continuously
It also supports load balancing, while updating UUID this package was dependent so in order to fix this bug, this package needs to be upgraded to its latest version

Reply:
it helps in making http request easier and while upgrading uuid this package was required to be upgraded 
since it has been deprecated, now axios are commonly used for the same 
but uuid was dependent on the same 
so thats why it was needed to upgrade it

Reply:
why would uuid depend on request package? uuid has nothing to do with http requests